pkg/nmstate/list.go

```diff
@@ -35,3 +37,30 @@ func ListPolicy(apiClient *clients.Settings) ([]*PolicyBuilder, error) {
 
 	return networkConfigurationPolicyObjects, nil
 }
+
+// ListNmStateConfig returns a NMStateConfig list.
+func ListNmStateConfig(apiClient *clients.Settings) ([]*NmStateConfigBuilder, error) {
```

Comment:
```suggestion
func ListNmStateConfigs(apiClient *clients.Settings, namespace string) ([]*NmStateConfigBuilder, error) {
```
Please include a namespace parameter for this function. If you are trying to get all nmstateconfigs across all namespaces intentionally, please add a separate function called ListNmStateConfigsInAllNamespaces